cavityFaultDisplay.py

```diff
@@ -74,6 +83,24 @@ def __init__(self, parent=None, args=None, macros=None):
             horizontalLayout.addWidget(shortDescriptionLabel)
             horizontalLayout.addWidget(statusLabel)
 
+            if fault.button_level == "GUI":
+                button = PyDMRelatedDisplayButton(filename=fault.button_command)
+
+            elif fault.button_level == "EDM":
+                button = PyDMEDMDisplayButton()
+                button.filenames = [fault.button_command]
+                button.macros = fault.macros + ',' + fault.button_macro
```

Comment:
This now successfully opens EDM panels to their specific tab. For panels that don't have a fault.button_macro (aka a column in the "Button Macros" cvs spreadsheet), will this cause any problems? With the way I implemented this, there will always be a ',' added onto button.macros. It doesn't look appear to have broken anything, but I just wanted to point it out

Reply:
Ok good to know! We can keep an eye on it! Alternatively, you can do 
```suggestion
                button.macros = fault.macros + (',' + fault.button_macro if fault.button_macro else "")
```

Reply:
Ooh, thank you!!! I was hoping you would have a clean coding fix and this is perfect, thank you:D

Reply:
Tested it on an OPI and it works:D